.github/workflows/macos.yml

```diff
@@ -1,18 +1,27 @@
-name: macOS
-
 on:
   push:
     branches:
       - master
   pull_request:
-  schedule:
```

Comment:
any reasons to remove the cron? I added it because it's the only way for me to see if `brew` makes a change breaking `dotbot-brew`. Like they did with `cask`. Since this repository doesn't get commits every day, I would love at least to run the tests every day to get a feedback if something breaks.

I am fine if some formulae is removed from `brew`, I can swap with another test target 😸

Reply:
Accidental, I copy pasted from my other copy and I may have missed that